.github/workflows/test-many-configs.yml

```diff
@@ -0,0 +1,35 @@
+
+name: auto-run-test-many-configs-script
+
+on: [push]
+
+
+jobs:
+  manyConfigsTest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
```

Comment:
Would be nice if we could configure exactly which version of Java 8, if that is even possible with this Action setup.

Reply:
From GH Actions documentation:
"The java-version input supports an exact version or a version range using SemVer notation:

major versions: 8, 11, 16, 17
more specific versions: 17.0, 11.0, 11.0.4, 8.0.232, 8.0.282+8
early access (EA) versions: 15-ea, 15.0.0-ea, 15.0.0-ea.2, 15.0.0+2-ea"

Out of the more specific versions, none of them match the java version I used on my local machine (8.0_292). Should I just leave it as the generic "java-version: '8' " ?